esmvaltool/diag_scripts/cmorizers/era5.py

```diff
@@ -0,0 +1,36 @@
+"""native6 diagnostic."""
+
+import logging
+import shutil
+from pathlib import Path
+
+from esmvaltool.diag_scripts.shared import (get_diagnostic_filename,
+                                            run_diagnostic)
+
+logger = logging.getLogger(Path(__file__).name)
+
+
+def main(cfg):
+    """Rename preprocessed native6 file."""
+    fixed_files = cfg['input_data']
+
+    for file, info in fixed_files.items():
+        stem = Path(file).stem
+        basename = stem.replace('native', 'OBS')
+
+        if info['diagnostic'] == 'daily':
+            basename = basename.replace('E1hr', 'Eday')
```

Comment:
```suggestion
            basename = basename.replace('E1hr', 'day')
```
But this probably needs some logic to look up the right mip table for the right variable. Any suggestions on how to do this most effictively, @bouweandela ?

Reply:
Now looking up the new mip in the cmor tables.